requirements.txt

```diff
@@ -1,8 +1,9 @@
 numpy>=1.7
 pandas>=0.13.1
-mhctools>=1.0.0
+mhctools>=1.3.0
 varcode>=0.3.17
 pylint>=1.4.4
 nose>=1.3.6
 gtfparse>=0.0.4
-mhcnames
\ No newline at end of file
+mhcnames
+pypandoc  # needs pandoc: brew install pandoc
```

Comment:
This seems different than what we usually do, e.g. https://github.com/hammerlab/pyensembl/blob/12f2fec8deb92bd86e990f26f4d1c983a23bd208/setup.py#L33

Would be good to have consistency here across the repos?